packages/office-ui-fabric-react/src/utilities/dragdrop/DragDropHelper.tsx

```diff
@@ -155,13 +164,20 @@ export class DragDropHelper implements IDragDropHelper {
         events.on(root, 'dragleave', onDragLeave);
         events.on(root, 'dragend', onDragEnd);
         events.on(root, 'drop', onDrop);
+        events.on(root, 'dragover', onDragOver);
       }
 
       if (isDraggable) {
         // If the target is draggable, wire up local event listeners for mouse events.
         onMouseDown = this._onMouseDown.bind(this, dragDropTarget);
+        onDragEnd = (event: DragEvent) => {
+          if (dragDropOptions.onDragEnd) {
+            dragDropOptions.onDragEnd(dragDropOptions.context.data, event);
+          }
+        };
```

Comment:
What do you mean by method on the class? We're trying to use the dragEnd function on the interface we passed it.